MIGRATE.md

```diff
@@ -1,15 +1,29 @@
-## webpack-migrate
-
-The `migrate` feature eases the transition from [version 1](http://webpack.github.io/docs/) to [version 2](https://gist.github.com/sokra/27b24881210b56bbaff7). `migrate`
-also allows users to switch to the new version of webpack without having to extensively [refactor](https://webpack.js.org/guides/migrating/).
-
-### Usage
-To use `migrate`, run the following command, with the value of `<config>` being a path to an existing webpack configuration file
+# webpack-cli migrate
+
+The `webpack-cli migrate` feature eases the transition from version 1 to version 2. `webpack-cli migrate`
```

Comment:
And from 2 to 4 (version 3 doesn't have breaking changes in terms of configuration)